### DISCUSSION TITLE
Week 1 Task 3

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#39] [Q&A]
Week 1 - Task 3, Assignment 5, Step 4, #3 and #4. 
I'm still struggling to get through some of week one's assignments. If someone could copy a picture of these steps, it would be helpful. I'm not seeing the options it wants me to click on.

**Comment by derekscholes** [comment]
I hope this is what you were talking about. I made a video of how to add suggestions. 

https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/132800749/d1b81bc5-a0a0-4c81-af7a-a7394e6eef7d
